src/renderer/window/window.cpp

```
@@ -72,16 +125,29 @@ vk::RenderingAttachmentInfoKHR mr::Window::target_image_info() noexcept
+
   std::array sems = {_render_finished_semaphore[image_index].get()};
   vk::PresentInfoKHR present_info {
     .waitSemaphoreCount = sems.size(),
     .pWaitSemaphores = sems.data(),
     .swapchainCount = 1,
-    .pSwapchains = &_swapchain._swapchain.get(),
+    .pSwapchains = (vk::SwapchainKHR*)&_swapchain._swapchain.swapchain,
     .pImageIndices = &image_index,
   };
-  _parent->vulkan_state().queue().presentKHR(present_info);
+
```

Cppcheck (2.19.0)

[style] The function 'add_camera' is never used.

(unusedFunction)